applications/nrf5340_audio/src/bluetooth/le_audio_bis_gateway.c

```diff
@@ -97,7 +97,7 @@ static void stream_sent_cb(struct bt_audio_stream *stream)
 	if (atomic_get(&iso_tx_pool_alloc[index])) {
 		atomic_dec(&iso_tx_pool_alloc[index]);
 	} else {
-		LOG_WRN("Decreasing atomic variable failed");
+		LOG_WRN("Decreasing atomic variable for stream %u failed", index);
```

Comment:
Is there a reason for using %u in the print, rather than %d? I believe we are using %d most places

Reply:
I dont think it is a specific reason. 
I think this also (in addition to the 0U comment above) is a result of the le_audio_(...) files being based on Zephyr samples.

Reply:
Maybe we should create a task for changing to "int i = 0" in for-loops, and changing %u to %d?

Reply:
> Maybe we should create a task for changing to "int i = 0" in for-loops, and changing %u to %d?

I created a task for this